### PR TITLE
Added new project, CivicTechJobs, to website

### LINKED ISSUE #2287: Add new project to website: Civic Tech Jobs
### Dependency
~Waiting on Civic Tech Jobs team to complete gathering of assets https://github.com/hackforla/CivicTechJobs/issues/36~

### Overview

We need to add a new project to the hack for la website so that our project list can be current

### Project Details
- Name: Civic Tech Jobs
- Name of future project.md file: civic-tech-jobs.md
- Files you will need to upload to the website repository: civic-tech-jobs.jpg and civic-tech-jobs-hero.jpg
- Where you will store the files:
   - images go in this folder: /assets/images/projects
   - project.md file goes in this folder: _projects

### Action Items
- [x] Take the content listed here and create a new project.md card (skip anything that has strikethrough)
- [x] Download the images
- [x] Add both project card and images to appropriate folders
- [x] Once you have it working on your local environment
  - [x] Take screen shots of the project card on the homepage and the projects project page adding them to issue
   - [x] Reach out to the product manager to ask if they are satisfied by adding a comment to [this issue](https://github.com/hackforla/CivicTechJobs/issues/36) with a link to the screenshots on your issue
- [x] Submit pull request
- [ ] Let Product Manager know when its live

### Resources
[Template for project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)

#### Content of the project.md card
- [x] A link to the [600 x 400 image](https://user-images.githubusercontent.com/37763229/149637447-2d336d29-e615-44df-82e4-b92a8ade7455.png)
- [x]  Alt image text - 'Hack for LA logo and the title of the project, Civic Tech Jobs'
- [x]  A link to the [1500 x 700 hero image](https://drive.google.com/file/d/1fCbocnGqLuw2fkKeAi3Ke0UVFo3qXaqE/view?usp=sharing) 
- [x]  Name of project: Civic Tech Jobs
- [x]  A blurb about your project:

> The CivicTechJobs.org MVP website will be a platform to help prospective volunteers find inter disciplinary projects that will be useful for their career development while contributing to positive civic impact and a CMS for Hack for LA projects to be able to list their open roles.
> 
> The tool will match volunteers by availability, role, and program area. Future iterations of the platform will focus helping volunteers find volunteer opportunities that match paid job postings, so that a volunteer can better prepare themselves for the marketplace.
- [x] Team members: (see comment below): https://github.com/hackforla/website/issues/2287#issuecomment-1019559014
- [x]  Links:
- GitHub: https://github.com/hackforla/civictechjobs
- Slack: [CivicTechJobs](https://hackforla.slack.com/messages/C02509WHFQQ)
- [x]  Any resources for a Getting Started link (a link to a wiki, readme(s) or both):
   - Wiki: 
   - Overview: [PDF](https://github.com/hackforla/product-management/blob/master/project-one-sheets/Civic-Tech-Jobs-One-Sheet.pdf)
- [x]  ~What you are looking for skills wise: N/A~
- [x]  ~Partner(s): N/A~
- [x]  Location: Remote
- [x]  Status: Active
- [x] Program Area: Civic Tech Infrastructure

### PR DESCRIPTION
**Pull request by Aveline-art**
Fixes #2287

Note to reviewer: Please let me know if any fields need to be escaped with quotations (' or "). Also, should I contact VRMS to update the meeting time for this project?

### What changes did you make and why did you make them ?

  - Added new project, Civic Tech Jobs, to the website
  - Resized the image slightly as they did not meet the 3:2 (600x400px) ratio specs
  - Added technologies and hero-alt text as per [this comment](https://github.com/hackforla/CivicTechJobs/issues/36#issuecomment-1022418452)
  - [Link to PM sign off](https://github.com/hackforla/CivicTechJobs/issues/36#issuecomment-1022483275)

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

Links to screenshot can be found [here ](https://github.com/hackforla/website/issues/2287#issuecomment-1022442053)in #2287

### CODE REVIEW

**Comment by SAUMILDHANKAR** [comment]
ETA: Eod 1/29/22
Availability: 3 hrs, 6-9 pm 1/29/22

File: _projects/civic-tech-jobs.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+description: The CivicTechJobs.org MVP website will be a platform to help prospective volunteers find inter disciplinary projects that will be useful for their career development while contributing to positive civic impact and a CMS for Hack for LA projects to be able to list their open roles.<br><br>The tool will match volunteers by availability, role, and program area. Future iterations of the platform will focus helping volunteers find volunteer opportunities that match paid job postings, so that a volunteer can better prepare themselves for the marketplace.
+image: /assets/images/projects/civic-tech-jobs.png
+alt: "Hack for LA logo and the title of the project, Civic Tech Jobs"
+image-hero: /assets/images/projects/civic-tech-jobs-hero.png
+alt-hero: "Civic tech jobs hero image, for HfLA website projects page"
+leadership: 
+  - name: Syed Danish Imran
+    role: Product Manager
+    links:
+      slack: U020Z27JB1S
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link: 
slack: https://hackforla.slack.com/team/U020Z27JB1S

File: _projects/civic-tech-jobs.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/sdimran
+    picture: https://avatars.githubusercontent.com/sdimran
+  - name: Sharad Gaurav
+    role: Product Manager
+    links:
+      slack: U01RETBMLKS
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U01RETBMLKS

File: _projects/civic-tech-jobs.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/SharadGaurav7
+    picture: https://avatars.githubusercontent.com/SharadGaurav7
+  - name: Ava Li
+    role: Full Stack Developer
+    links:
+      slack: U01B73XGCKV
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U01B73XGCKV

File: _projects/civic-tech-jobs.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/Aveline-art
+    picture: https://avatars.githubusercontent.com/Aveline-art
+  - name: Sarah Williams
+    role: Software Developer
+    links:
+      slack: U01S1S75NLW
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U01S1S75NLW

File: _projects/civic-tech-jobs.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/arghmatey
+    picture: https://avatars.githubusercontent.com/arghmatey
+  - name: Jen Chung
+    role: UX/UI Designer
+    links:
+      slack: U02A6H5PVAA
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U02A6H5PVAA

File: _projects/civic-tech-jobs.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/jenchuu
+    picture: https://avatars.githubusercontent.com/jenchuu
+  - name: Grace Lin
+    role: UX/UI Designer
+    links:
+      slack: U02AE5H5BJ9
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U02AE5H5BJ9

File: _projects/civic-tech-jobs.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/graceerya
+    picture: https://avatars.githubusercontent.com/graceerya
+  - name: Patty Costello
+    role: UX Researcher
+    links:
+      slack: U01UTFC17L4
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U01UTFC17L4

File: _projects/civic-tech-jobs.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/patula006
+    picture: https://avatars.githubusercontent.com/patula006
+  - name: Stephanie G. Stahlberg
+    role: UX Researcher
+    links:
+      slack: U02H06YBR7F
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U02H06YBR7F

File: _projects/civic-tech-jobs.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/stephaniestahlberg
+    picture: https://avatars.githubusercontent.com/stephaniestahlberg
+  - name: Helene Dendor
+    role: UX Researcher
+    links:
+      slack: U02C41H7919
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: Correct slack link:
slack: https://hackforla.slack.com/team/U02C41H7919

File: _projects/civic-tech-jobs.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+      github: https://github.com/dendorable
+    picture: https://avatars.githubusercontent.com/dendorable
+links: 
+  - name: Github
+    url: https://github.com/hackforla/civictechjobs
+  - name: Slack
+    url: https://hackforla.slack.com/messages/C02509WHFQQ
+  - name: Overview
+    url: https://github.com/hackforla/product-management/blob/master/project-one-sheets/Civic-Tech-Jobs-One-Sheet.pdf
```
Comment (SAUMILDHANKAR) [inline _projects/civic-tech-jobs.md]: The pdf should be placed in this folder: https://github.com/hackforla/website/tree/gh-pages/assets/pdfs
Currently, the pdf does not open in full page.

**Comment by SAUMILDHANKAR** [comment]
@Aveline-art Thank you working on this issue. I am not sure about reaching out to VRMS, please let me know if you want a new issue for that. Thanks.

**Comment by Aveline-art** [comment]
@SAUMILDHANKAR Thanks for the review! I've made all the necessary corrections. I think the team might have talked about this already, but just to be sure, I accidentally discovered that the one-sheet pdf viewer can be accessed when downloading it, as with [this](https://raw.githubusercontent.com/hackforla/product-management/master/project-one-sheets/Civic-Tech-Jobs-One-Sheet.pdf).

As for the meeting times, I will speak with the CTJ team and handle it from there. Thanks for getting back to me!

**Review by SAUMILDHANKAR — COMMENTED** [review]
@Aveline-art Thanks for making the changes. All looks good. I might have missed it during the meeting, but I am not aware of one-sheet viewer. When I clicked on your link, just downloaded the pdf sheet. Also, on the projects page I still see the overview link as https://github.com/hackforla/product-management/blob/master/project-one-sheets/Civic-Tech-Jobs-One-Sheet.pdf, whereas on CTJ page it has been changed and opens as a pdf. I see it is different in some other projects as well so we can ignore it for now and deal with it separately.

Thanks for editing upper case H in GitHub name, my bad I missed it. I also wanted to let you know that the slack channel link for CTJ currently is https://hackforla.slack.com/messages/C02509WHFQQ whereas for all other projects the format is https://hackforla.slack.com/archives/C018S5TCQE7 (notice the difference between messages and archives). Both are redirecting fine but just wondered if it might be an issue in future.
So, please let me know your comments. I will merge it once I hear back. Thank you.

**Comment by Aveline-art** [comment]
> I still see the overview link as https://github.com/hackforla/product-management/blob/master/project-one-sheets/Civic-Tech-Jobs-One-Sheet.pdf, whereas on CTJ page it has been changed and opens as a pdf.

I suspect that your cache might not be cleared since this does not seem to be happening [on my PC](https://user-images.githubusercontent.com/62368440/151718915-edaddd1d-1f0a-4f82-a53b-d1470f5ed5d3.png).

For the Slack link, I'm not sure if there is a difference with each url... I took it from the original issue. Please advise on the best way to proceed with it.

**Comment by SAUMILDHANKAR** [comment]
@Aveline-art You are right, I stand corrected. Overview link is fine, was cache issue on my side. All looks good now.

I have noted the differences for slack link, but since it is not related to this issue. I will go ahead and merge this. Thank you for all the clarifications.

**Review by SAUMILDHANKAR — APPROVED** [review]

**Pull request merged**